packages/playwright/bundles/babel/src/babelBundleImpl.ts

```diff
@@ -23,7 +23,6 @@ import * as babel from '@babel/core';
 export { codeFrameColumns } from '@babel/code-frame';
 export { declare } from '@babel/helper-plugin-utils';
 export { types } from '@babel/core';
-export { parse } from '@babel/parser';
```

Comment:
I think we can drop it from the package.json again?

Reply:
This package is a dependency of babel/core, we can remove it, add it, then remove it again, but nothing will change.

Reply:
Better to let NPM handle the version matching rather than us? I think thats my main argument + we try to not have packages which we don't require ourselves inside the package.json

Reply:
I agree in principle. But here is out list:

```js
  "dependencies": {
    "@babel/code-frame": "^7.24.2",
    "@babel/core": "^7.24.4",
    "@babel/helper-plugin-utils": "^7.24.0",
    "@babel/parser": "^7.24.4",
    "@babel/plugin-proposal-decorators": "^7.24.1",
    "@babel/plugin-proposal-explicit-resource-management": "^7.24.1",
    "@babel/plugin-syntax-async-generators": "^7.8.4",
    "@babel/plugin-syntax-import-attributes": "^7.24.1",
    "@babel/plugin-syntax-json-strings": "^7.8.3",
    "@babel/plugin-syntax-object-rest-spread": "^7.8.3",
    "@babel/plugin-syntax-optional-catch-binding": "^7.8.3",
    "@babel/plugin-transform-class-properties": "^7.24.1",
    "@babel/plugin-transform-class-static-block": "^7.24.4",
    "@babel/plugin-transform-export-namespace-from": "^7.24.1",
    "@babel/plugin-transform-logical-assignment-operators": "^7.24.1",
    "@babel/plugin-transform-modules-commonjs": "^7.24.1",
    "@babel/plugin-transform-nullish-coalescing-operator": "^7.24.1",
    "@babel/plugin-transform-numeric-separator": "^7.24.1",
    "@babel/plugin-transform-optional-chaining": "^7.24.1",
    "@babel/plugin-transform-private-methods": "^7.24.1",
    "@babel/plugin-transform-private-property-in-object": "^7.24.1",
    "@babel/plugin-transform-react-jsx": "^7.23.4",
    "@babel/preset-typescript": "^7.24.1"
  },
  "devDependencies": {
    "@types/babel__code-frame": "^7.0.4",
    "@types/babel__core": "^7.20.2",
    "@types/babel__helper-plugin-utils": "^7.10.1",
    "@types/babel__traverse": "^7.20.2"
  }
```